tests/longhaul/block_test.go

```diff
@@ -28,12 +29,18 @@ type BlockLongHaulSuite struct {
 	suite.Suite
 	kh        *utils.K8sHelper
 	installer *installer.InstallHelper
+	op        contracts.Setup
 }
 
 //Test set up - does the following in order
 //create pool and storage class, create a PVC, Create a MySQL app/service that uses pvc
 func (s *BlockLongHaulSuite) SetupSuite() {
-	s.kh, s.installer = setUpRookAndPoolInNamespace(s.T, "longhaul-ns", "rook-block", "rook-pool")
+	o := NewBaseLoadTestOperations(s.T, "longhaul-ns")
```

Comment:
could you make the same change to the `NewBaseLoadTestOperations()` as you did for `NewBaseTestOperations()`?

Reply:
This is done now.